src/Relation/Nullary/Construct/Add/Point.agda

```diff
@@ -8,8 +8,84 @@
 
 module Relation.Nullary.Construct.Add.Point where
 
+open import Algebra.Bundles
```

Comment:
This content should be in a new module `Algebra.Construct.Add.Point` rather than here I think!